site/pages/gcweb-theme/static-header-footer/bootstrap-4.html

```diff
@@ -0,0 +1,122 @@
+	<p>Nulla tempus odio fringilla dui egestas, sed hendrerit nunc tempor. Donec ultricies nulla leo, eget fringilla libero lobortis tristique. Curabitur pellentesque luctus odio, vitae semper ex. Integer posuere sem est, in hendrerit sem cursus ut. Aliquam felis felis, sollicitudin eu tincidunt non, egestas et orci.</p>
+	<p><a href="bootstrap-3.html" class="btn btn-primary">Switch to Bootstrap 3</a></p>
+	<div class="pagedetails">
+		<dl id="wb-dtmd">
+				<dt>Date modified:</dt>
+				<dd><time>2019-09-12</time></dd>
+		</dl>
+	</div>
+</main>
+
+<!-- Static footer starts -->
+<footer id="wb-info" class="footer-spacing">
+	<div class="landscape">
+		<nav class="container wb-navcurr">
+			<h2 class="sr-only">About government</h2>
+			<ul class="list-unstyled colcount-sm-2 colcount-md-3">
+				<li><a href="https://www.canada.ca/en/contact.html">Contact us</a></li>
+				<li><a href="https://www.canada.ca/en/government/dept.html">Departments and agencies</a></li>
+				<li><a href="https://www.canada.ca/en/government/publicservice.html">Public service and military</a></li>
+				<li><a href="https://www.canada.ca/en/news.html">News</a></li>
+				<li><a href="https://www.canada.ca/en/government/system/laws.html">Treaties, laws and regulations</a></li>
+				<li><a href="https://www.canada.ca/en/transparency/reporting.html">Government-wide reporting</a></li>
+				<li><a href="https://pm.gc.ca/eng">Prime Minister</a></li>
+				<li><a href="https://www.canada.ca/en/government/system.html">How government works</a></li>
+				<li><a href="https://open.canada.ca/en/">Open government</a></li>
+			</ul>
+		</nav>
+	</div>
+	<div class="brand">
+		<div class="container">
+			<div class="row">
+				<nav class="col-lg-9 col-xl-10 ftr-urlt-lnk">
+					<h2 class="sr-only">About this site</h2>
+					<ul class="list-inline">
+						<li class="list-inline-item"><a href="https://www.canada.ca/en/social.html">Social media</a></li>
+						<li class="list-inline-item"><a href="https://www.canada.ca/en/mobile.html">Mobile applications</a></li>
+						<li class="list-inline-item"><a href="https://www1.canada.ca/en/newsite.html">About Canada.ca</a></li>
+						<li class="list-inline-item"><a href="https://www.canada.ca/en/transparency/terms.html">Terms and conditions</a></li>
+						<li class="list-inline-item"><a href="https://www.canada.ca/en/transparency/privacy.html">Privacy</a></li>
+					</ul>
+				</nav>
+				<div class="col-6 offset-6 offset-lg-0 col-lg-3 col-xl-2 text-right"> <img src="https://www.canada.ca/etc/designs/canada/wet-boew/assets/wmms-blk.svg" alt="Symbol of the Government of Canada"> </div>
+			</div>
+		</div>
+	</div>
+</footer>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.js"></script>
```

Comment:
Is jQuery needed in this Bootstrap 4 experimental page?

Reply:
No it is not, I'll remove it!